docs/pages/setup/security/reduce-blast-radius.mdx

```diff
@@ -0,0 +1,134 @@
+
+For clusters that use the [local authentication connector](/docs/setup/reference/authentication/#local), Teleport lets you make it mandatory for a user to enroll an MFA device when they create an account, and to authenticate using that device when they begin a new Teleport session. To do so, add the following to your Teleport [configuration file](/docs/setup/reference/config):
+
+```yaml
+auth_service:
+
+    authentication:
+        type: local
+        second_factor: <factor type>
+```
+
+To make MFA mandatory for all users, `second_factor` must be set to one of the following values:
+
+- `otp`
+- `u2f`
+- `webauthn`
+- `on`
+
+Choose `on` if you would like to require MFA for all users while letting them choose an OTP, U2F, or WebAuthn device. The other options restrict users to a single type of MFA device, which is useful for enforcing a particular standard of security. Once you start the Teleport [proxy service](/docs/architecture/proxy/) with the `second_factor` configuration option set to one of these values, Teleport will mandate MFA by:
+
+- Adjusting the Teleport [signup page](/docs/getting-started/linux-server/#step-24-create-a-teleport-user-and-set-up-two-factor-authentication) so a user must enroll an MFA device of the kind you have selected. If the value of `second_factor` is `on`, users will have the option to select from multiple device types.
+
+- Presenting the user with an MFA challenge when they run `tsh login`
+
+For authentication connectors besides `local`, you must make MFA mandatory via your identity provider. For example, Teleport clusters using [GitHub for SSO](/docs/setup/admin/github-sso/) can require users in their organization to enroll an MFA device by adjusting their [organization-level settings](https://docs.github.com/en/organizations/keeping-your-organization-secure/requiring-two-factor-authentication-in-your-organization).
+
+
+## Present an MFA challenge for every attempt to access a resource
+After a user logs into a Teleport cluster, they can request access to a particular resource, e.g., a node, database, application, or Kubernetes cluster. In this case, the Teleport [Auth Service](/docs/architecture/authentication/) issues a single-use certificate for accessing that resource. You can prevent attackers from doing damage with a compromised certificate by enabling [per-session MFA](/docs/access-controls/guides/per-session-mfa/). 
+
+With this setting, whenever a user requests a one-time certificate to access a resource, the Teleport Auth Service will issue an MFA challenge, even if the user has already begun a Teleport session via `tsh login`. To enable per-session MFA for all users, make the following change to your Teleport configuration file:
+
+```yaml
```

Comment:
nit: this is only possible for users in the self-hosted mode. Let's use details with scope and tabs with scope to have references for cloud and self hosted. Cloud users can use RBAC options to achieve this effect.

Reply:
👍